dev/build.image/build.gradle

```diff
@@ -673,3 +673,81 @@ task createJSONForPublicArtifacts {
 task gatherTestResults {
     dependsOn zipTestReport,createJSONForPublicArtifacts
 }
+
+task copyOpenLiberty(type: Sync) {
+   dependsOn zipOpenLiberty
+
+   mkdir("packaging/tempPackagingDir")
+
+   from zipOpenLiberty
+   into 'packaging/tempPackagingDir'
+}
+
+task updateSpecTemplate(type: Copy) {
+    from 'packaging'
+    include 'openliberty.spec.template'
+    into 'packaging/rpmbuild/SPECS'
+    filter { line -> line.replaceAll('@SPEC_VERSION@', bnd('libertyServiceVersion', '19.0.0.5')) }
```

Comment:
Is 19.0.0.5 a default if you can't determine the real version? Might be better to default to "Unknown" rather than guess incorrectly.